src/metabase/related.clj

```diff
@@ -93,14 +94,14 @@
   (let [[best rest] (split-at max-best-matches matches)]
     (concat best (->> rest shuffle (take max-serendipity-matches)))))
 
-(def ^:private ^{:arglists '([entities])} filter-visible
-  (partial filter (fn [{:keys [archived visibility_type active] :as entity}]
-                    (and (some? entity)
+(def ^:private ^{:arglists '([instances])} filter-visible
```

Comment:
Our terminology here is a bit all over the place. *instances* of a Toucan *model* are what we're using in most places, and what Toucan 2 uses, so I updated some stuff to match that terminology.